src/azure-cli-core/azure/cli/core/commands/__init__.py

```diff
@@ -397,25 +397,24 @@ def _execute_command(kwargs):
             op = get_op_handler(operation)
             try:
                 result = op(client, **kwargs) if client else op(**kwargs)
+                if no_wait_param and kwargs.get(no_wait_param, None):
+                    return None  # return None for 'no-wait'
```

Comment:
@tjprescott Think you recently create a PR for a transform for the `--no-wait` param.
Would this change mean that that transform logic is no longer needed?